### DISCUSSION TITLE
I have problem when I've tried to use the red close tab button from CustomCSSforFx

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#705] [General]
I love Firefox UI Fix, and I also love CustomCSSforFx, however I have problem: 
- When I tried to add import to my userChrome.css, and I wanted to import the `close_icon_red.css` from CustomCSSforFx, then the red close tab button appears incorrectly, so can you add compatibility for CustomCSSforFx's red close tab button please?

Here's my import value from userChrome.css:

> @import "./css/generalui/close_icon_red.css";

And I'm attaching a screenshot with red close tab button enabled (Well, it is not displayed correctly):
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/d391a74c-7a04-4d46-86b0-1e19de6a3b9f)

Here's how the red close tab button should look like (Screenshot from Basilisk):
![Original red close tab button](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/2dfc0f49-5e7a-4980-b6c2-f21cb7a61824)

So, I will be very happy if you investigate the issue, and then add a compatibility to Firefox UI Fix for allowing to use CustomCSSforFx's red close tab button. Thank you, sir.

**Comment by Aleksandra0056** [comment]
Well, problem is, that the Firefox UI Fix is surely causing this, because I've tried in Virtual Machine without Firefox UI Fix, and then it was displayed correctly, and I've installed Firefox UI Fix in Virtual Machine, then it was broken.

**Comment by black7375** [comment]
Mixing it with other projects can break the UI and is not guaranteed to work correctly.
It takes a lot of resources just to maintain compatibility between Firefox versions, some add-ons, and different OS and browser settings.

**Reply by Aleksandra0056** [reply]
Thank you anyway, but can you at least find what is causing it?

**Comment by Aleksandra0056** [comment]
Yes!!! I found what was causing this problem! I have to simply turn the `userChrome.tab.close_button_at_hover` to `false` from the `user.js` through Notepad++

![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/83855fb7-5397-4723-add8-1a994375a0ad)
